homeworks/lesia.turua_lesiaturuta/PokemonProject/scripts/pokemon.js

```diff
@@ -315,6 +66,34 @@ const cleanElement = (el) => {
     }
 };
 
+const renderSelect = () => {
+    cleanElement(selectSearchEl);
+    const optionFirstEl = document.createElement('option');
+    optionFirstEl.value = 'Select type: ';
+    optionFirstEl.text = 'Select type: ';
+    optionFirstEl.selected = true;
+    optionFirstEl.disabled = true;
+    selectSearchEl.add(optionFirstEl);
+    typesPokemons.forEach((type) => {
+        const optionEl = document.createElement('option');
+        optionEl.value = type;
+        optionEl.text = type;
+        selectSearchEl.add(optionEl);
+    });
+};
+
+const sortPokemonsUp = (array) => array.sort((a, b) => a.height - b.height);
+
+const sortPokemonsDown = (array) => array.sort((a, b) => b.height - a.height);
+
+const updateTypes = (array) => {
```

Comment:
If a function returns something, better to show it in its name: `getPokemonTypes`.
 And an `array` is too common name for this parameter. If you are working with pokemons, name it `pokemons`

Reply:
still need to fix